react-commenting.js

```diff
@@ -43,6 +46,7 @@ class Board extends Component{
     }
     if(winner === null){
       // 5) Here:
+      //Set the winning state for the winner in these square's values
       this.winning()
```

Comment:
This line is calling the method called winning. The winning method will update the state, but this line is just calling the method as long as there is not a winner yet.